Rollbar/RollbarConfiguration.m

```diff
@@ -100,6 +110,17 @@ - (void)setServerHost:(NSString *)host root:(NSString*)root branch:(NSString*)br
     [self save];
 }
 
+- (void)setNotifierName:(NSString *)name version:(NSString *)version {
+    self.notifierName = name ? name : NOTIFIER_NAME;
+    self.notifierVersion = version ? version : NOTIFIER_VERSION;
+    [self save];
+}
+
+- (void)setFramework:(NSString *)framework {
```

Comment:
Do we also need this for version or are we using the version from this SDK?

Reply:
It is at Line 113 above

Reply:
Ah, I missed the param. Thanks